target_chains/cosmwasm/tools/src/chains-manager/chain-querier.ts

```diff
@@ -0,0 +1,102 @@
+import { Tendermint34Client } from "@cosmjs/tendermint-rpc";
+import { QueryClient } from "@cosmjs/stargate";
+import { WasmExtension, setupWasmExtension } from "@cosmjs/cosmwasm-stargate";
+
+export type ContractInfoRequest = {
+  contractAddr: string;
+};
+
+export type ContractInfoResponse = {
+  codeId: number;
+  creator: string;
+  admin: string;
+  label: string;
+};
+
+export type SmartContractRequest = {
+  contractAddr: string;
+  query: Object;
+};
+
+export type RawContractStateRequest = {
+  contractAddr: string;
+  key: Buffer;
+};
+
+export type AllContractStateRequest = {
+  contractAddr: string;
+};
+
+export interface ChainQuerier {
```

Comment:
(this is a nitpick) I think this code would be easier to understand if you defined this interface first in the file, then had the request types after. This interface is the main thing the reader needs to understand, and once they see this, the request types will make sense. (similar organizational comment applies to the file above)

also please document these methods, especially since they return `Object` which doesn't say a lot about what's in them.